python/tvm/script/ir_builder/tir/ir.py

```diff
@@ -1782,6 +1798,11 @@ def wrapped(*args, **kwargs):
 tvm_bmma_sync = _op_wrapper(_tir_op.tvm_bmma_sync)
 tvm_fill_fragment = _op_wrapper(_tir_op.tvm_fill_fragment)
 tvm_store_matrix_sync = _op_wrapper(_tir_op.tvm_store_matrix_sync)
+tvm_storage_sync = _op_wrapper(_tir_op.tvm_storage_sync)
```

Comment:
No need to use op_wrapper - this is only used for backward compatibility

Reply:
removed.